prebuilt-ui/basic-embed/components/Header.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+
+export const Header = () => (
```

Comment:
this header seems like a good thing to put in the shared components and then we can pass in a title and github link as props. Not necessary but might be nice

Reply:
First pass done! I am excited for it to eventually be Steve-ified.